### ISSUE TITLE
Patch symbol error in toolchain.py in Kivy-iOS

### ISSUE THREAD

**Issue by mamunabcoder** [kivy/kivy-ios#646]
You can install kivy-ios from master by typing the following:
```
pip install https://github.com/kivy/kivy-ios/archive/refs/heads/master.zip
```

_Originally posted by @mawaki in https://github.com/kivy/kivy-ios/issues/629#issuecomment-907291824_

Versions

Python : 3.9.6
MacOS version : Catalina
XCode Version : 12.4
Cython version : 0.29.24

3 days ago updated the master branch then found below issue to Kivy-ios

I have found another issue when try to install:
pip install https://github.com/kivy/kivy-ios/archive/refs/heads/master.zip

Below error found:

RAN: /usr/bin/patch -t -d /Users/bjitlimited/kivy_rnd/build/hostlibffi/x86_64/libffi-3.2.1 -p1 -i /Users/bjitlimited/kivy_rnd/kivy_venv/lib/python3.9/site-packages/kivy_ios/recipes/hostlibffi/fix-win32-unreferenced-symbol.patch

  STDOUT:

  STDERR:
/usr/bin/patch: **** Can't open patch file /Users/bjitlimited/kivy_rnd/kivy_venv/lib/python3.9/site-packages/kivy_ios/recipes/hostlibffi/fix-win32-unreferenced-symbol.patch : No such file or directory

**Comment by ricrudo** [comment]
Same problem here!!

**Comment by misl6** [comment]
The issue should be fixed on current master via #651, can you verify if we can mark the issue as fixed and report back the result?

**Comment by misl6** [comment]
Closing due to inactivity.
(Issue should be fixed via #651)
